### PR TITLE
Fix the output of format (whitespace-only) blocks inside inline partials

### LINKED ISSUE #556: Dust.js has started outputting 'line,2 line,4 line,13 line,6' in weird places
Come DustJS 2.6.0 it has started outputting line,[number] in random places. It seems to be affected by the `dust.config.whitespace` setting. Weirdly enough I got the unwanted output when `dust.config.whitespace = false` on a DO droplet (Node v0.10.33), while I got it when `dust.config.whitespace = true` on my workstation. Downgrading to v2.5.1 resolved the problem.

I precompile all my templates.

I am pressed for time the coming weeks, but if nobody else is experiencing this I could dig some more to find out exactly where the error occurs.

### PR DESCRIPTION
**Pull request by sethkinast**
Closes #556

### CODE REVIEW

File: lib/compiler.js [hunk 1]
```diff
@@ -233,7 +233,7 @@
     },
 
     format: function(context, node) {
-      return '.w(' + escape(node[1] + node[2]) + ')';
+      return '.w(' + escape(node[1]) + ')';
```
Comment (smfoote) [inline lib/compiler.js]: I really hate that our parser gives us things like `node[0]` and `node[1]` instead of `node.body` and `node.lineNo`. Tornado...

**Comment by smfoote** [comment]
Looks good to me.

File: lib/compiler.js (hunk 1 above)
Comment (jimmyhchan) [inline lib/compiler.js]: it's done for perf... (supposedly)

**Comment by jimmyhchan** [comment]
looks good + 1

**Pull request merged**